adoc/SAP-S4HA10-setup-simplemount-sle15.adoc

```diff
@@ -642,8 +642,9 @@ For the ERS and ASCS instances, edit the instance profiles
 {mySid}_{myInstAscs}_{myVipNAscs} and {mySid}_{myInstErs}_{myVipNErs} in the
 profile directory _/usr/sap/{mySid}/SYS/profile/_.
 
-Tell the `{sapStartSrv}` service to load the HA script connector library and
-to use the connector `{s4sClConnector3}`.
+Tell the `{sapStartSrv}` service to load the HA script connector library and to
+use the connector `{s4sClConnector3}`. On the other hand, please make sure the
+feature _Autostart_ is *not* used.
```

Comment:
```
On the other hand, please make sure the
feature _Autostart_ is *not* used.
```
should be: 

Additionally, make sure that the feature _Autostart_ is *not* used.